### PR TITLE
Add grape js builder bundle.

### PR DESCRIPTION
**Pull request by mohit-rocks**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               |no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | @todo
| Related developer documentation PR URL | @todo 
| Issue(s) addressed                     | Fixes #

#### Description:
Adding 1.0-RC(https://github.com/mautic/plugin-grapesjs-builder/releases/tag/1.0-RC) release for Mautic 3

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -112,7 +112,9 @@
         "gaufrette/extras": "^0.1.0",
         "gaufrette/aws-s3-adapter": "^0.4.0",
         "phpoffice/phpspreadsheet": "^1.15",
-        "predis/predis": "^1.1"
+        "predis/predis": "^1.1",
+        "composer/installers": "^1.10",
+        "mautic/grapejs-builder": "1.0-RC"
```
Comment (RCheesley) [inline composer.json:117]:
```suggestion
        "mautic/GrapesJsBuilderBundle": "1.0-RC"
```

File: composer.json [hunk 2]
```diff
@@ -139,7 +141,24 @@
         {
             "type": "git",
             "url": "https://github.com/mautic/intl.git"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "mautic/grapejs-builder",
```
Comment (RCheesley) [inline composer.json:148]:
```suggestion
                "name": "mautic/GrapesJsBuilderBundle",
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
I think this is why the tests are failing?

File: composer.json (hunk 2 above)
Comment (mohit-rocks) [reply composer.json:148]: Added following snippet.
```
"extra": {
                    "installer-name": "GrapesJsBuilderBundle"
                }
```
Let's see how it goes.

**Review by RCheesley — APPROVED** [review]
That was it - plugin is now showing:

screenshot-mautic ddev site-2021 02 15-18_55_57

and I can enable it in Mautic and work with supported templates!

screenshot-mautic ddev site-2021 02 15-19_00_13

**Review by kuzmany — APPROVED** [review]
Works for me 👍

**Comment by StrategieSociali** [comment]
Works for me.
I'm testing landing pages. Mobile views has some problems. I'm not a grapesjs guru but it seems that when a put, for eg an image, and set this parameters:
Image1: display->none in mobile views
Image2: display->none in desktop views
If I come back in desktop views and then again in mobile view all the images disappear.
If I publish the landing and see it in mobile no images are display. In desktop I can see only the desktop image (correct).
The same actions in grapesjs.com work fine, no issues.

**Comment by jonathandhn** [comment]
I've just upgraded 3 instances of Mautic to Mautic 3.3, one with grapjs RC4 already installed (From Mautic 3.3 RC) & the two others without the plugin (from Mautic 3.2.5), prior to the upgrade.  GrapeJS is there for where it used to be, GrapeJS isn't there where is supposed to be bundled (Plugin folder)

GrapeJS isn't in the bundle released today, on the contrary of the change log published for Mautic 3.3, is that a Bug ? Or do I don't understand the purpose of this ticket ? 

Changelog of 3.3 https://github.com/mautic/mautic/commit/692f0d922889b296208723056056a76f4c3b87a4
Enhancements :

- Add grape js builder bundle. #9677
- Support for builder specific templates #9654

EDIT : 

I've got it download both, unzip, go to plugin folder. GrapeJS is only on one of the two zip file.
https://github.com/mautic/mautic/releases/download/3.3.0/3.3.0-update.zip
https://github.com/mautic/mautic/releases/download/3.3.0/3.3.0.zip

**Pull request merged**